client/src/views/auth/signIn.tsx

```diff
@@ -1,14 +1,43 @@
 import React from "react"
-import Button from '@mui/material/Button'
+import Button from '@mui/material/Button';
+import TextField from '@mui/material/TextField';
+import Box from '@mui/material/Box';
+const image_disp = require("../../image/doctor1.png").default;
+const logo = require("../../image/logo.png").default;
+import Visibility from '@mui/icons-material/Visibility';
+import VisibilityOff from '@mui/icons-material/VisibilityOff';
 
 const SignIn: React.FC = () => {
-    return (
-        <div>
-            <h1 className="has-text-primary">
-                this is signin page
-            </h1>
-            <Button  color="success" variant="contained">Hello world</Button>
-        </div>
+    return (  
+        <Box 
+            component="form"
+            sx={{
+                
+                '& .MuiTextField-root': { m: 0.5,  width: '30ch' },
+                '& button': { m: 1, width: '28ch' } ,     
+            }}
+            noValidate
+            autoComplete="off"
+          
+        >
+            <div></div>
+            <div className="box">
+                {/* LOGIN CREDENTIALS */}
+                <center>
+                    <img className="imgLogo" src={logo} alt="logo"></img><br></br><br></br>
+                    <TextField required color="primary"  variant="outlined" label="Email" type="email" /><br></br>
+                    <TextField required variant="outlined" label="Password" type="password" /><br></br>
```

Comment:
create a component for textField save to directory views/components

and just reuse it into this page and pass the label, value, onChange, type as props

and do the same thing with the button